src/Projection/Query.php

```diff
@@ -29,6 +30,8 @@ public function fromStream(string $streamName): Query;
 
     public function fromStreams(string ...$streamNames): Query;
 
+    public function fromStreamWithMetadataMatcher(string $streamName, MetadataMatcher $metadataMatcher): Query;
```

Comment:
This is a BC break and should be released in a new major version, shouldn't it?

Reply:
Exactly, my bad. I think this should work:
`public function fromStream(string $streamName, MetadataMatcher $metadataMatcher = null): Query;`